README.md

```diff
@@ -1,10 +1,18 @@
 # PHP Docker Images
 
-These images are built directly from source with PHP extensions in use on the WP Engine Platform.
+These docker images provide each major version of PHP (currently 5.6, 7.0 and 7.1) with support for various extensions required for WordPress and the WP Engine Platform.
+
+There are two main groups of images provided here:
+ - Alpine
+ - Busybox
+
+The Alpine images are based on the precompiled php-fpm-alpine images available at https://hub.docker.com/_/php/. The compiler used in Alpine is `musl` (instead of glibc) and all extensions used must be able to compile using musl as well. Currently, ionCube does not provibe a musl-based extension which created the need for an alternative set of images using glibc.
+
+The Busybox images are compiled from source downloaded from PHP.net. The compiler used is glibc which allows the ionCube extension (and any other glibc-only extensions) to work.
```

Comment:
The compiler is still GCC, glibc is the C library that we compile aginst.

Reply:
Ah...good catch.